exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/WorkerThread.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.exporter.stats.stackdriver;
+
+import com.google.cloud.monitoring.v3.MetricServiceClient;
+import com.google.common.collect.Lists;
+import com.google.monitoring.v3.CreateTimeSeriesRequest;
+import io.opencensus.common.Duration;
+import io.opencensus.stats.View;
+import io.opencensus.stats.ViewData;
+import io.opencensus.stats.ViewManager;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.concurrent.GuardedBy;
+
+/**
+ * Worker {@code Thread} that polls ViewData from Stats library and batch export to StackDriver.
+ *
+ * <p>{@code WorkerThread} is a daemon {@code Thread}.
+ */
+final class WorkerThread extends Thread {
+
+  private static final Object monitor = new Object();
+
+  private final long scheduleDelayMillis;
+  private final String projectId;
+  private final MetricServiceClient metricServiceClient;
+
+  @GuardedBy("monitor")
+  private final ViewManager viewManager;
+
+  @GuardedBy("monitor")
+  private final Map<View.Name, View> registeredViews = new HashMap<View.Name, View>();
+
+  WorkerThread(
+      String projectId,
+      MetricServiceClient metricServiceClient,
+      Duration exportInterval,
+      ViewManager viewManager) {
+    this.scheduleDelayMillis = toMillis(exportInterval);
+    this.projectId = projectId;
+    this.metricServiceClient = metricServiceClient;
+    this.viewManager = viewManager;
+    setDaemon(true);
+    setName("ExportWorkerThread");
```

Comment:
Consider "StackdriverExporterWorkerThread" in case there are ever other exporters.

Reply:
Done.